tests/validatorsDumps.ts

```diff
@@ -0,0 +1,62 @@
+import { Connection } from '@solana/web3.js';
+import { deserializeUnchecked, serialize } from 'borsh';
+import BN from 'bn.js';
+
+import fs from 'fs';
+import path from 'path';
+
+import { AccountList, validatorsSchema } from '@/unstake';
+import { VALIDATOR_LIST } from '@/constants';
+import { ValidatorsList } from '@/types';
+
+const connection = new Connection(
+  'https://pyth-testnet-rpc-1.solana.p2p.org/yIwMoknPihQvrhSyxafcHvsAqkOE7KKrBUpplM5Xf',
+);
+
+const serializeAndSaveToFile = async (deserializedValidators, namePostFix: string) => {
+  const serializedValidators = serialize(validatorsSchema, deserializedValidators);
+  //@ts-ignore
+  const { data } = serializedValidators.toJSON();
+
+  const validatorsListJson = JSON.stringify({ data });
+
+  const filename = `validator_list${namePostFix}.json`;
+  fs.writeFileSync(path.join(__dirname, 'data', filename), validatorsListJson, 'utf-8');
+
+  console.log(`Done ${filename}`);
+};
+
+const updateValidatorListDump = async () => {
+  const validatorListResp = await connection.getAccountInfo(VALIDATOR_LIST);
+
+  if (validatorListResp === null) {
+    console.log('getAccountInfo of validatorList returned null, try again');
+    return;
+  }
+
+  const deserializedValidators = deserializeUnchecked(
+    validatorsSchema,
+    AccountList,
+    validatorListResp.data,
+  ) as any as ValidatorsList;
```

Comment:
Why just  `) as ValidatorsList;` is not enough? As I can see no errors appear after this change